go/roothash/api/commitment/executor.go

```diff
@@ -151,10 +164,17 @@ func (m *ComputeBody) ValidateBasic() error {
 		if header.StateRoot != nil {
 			return fmt.Errorf("failure indicating commitment includes StateRoot")
 		}
+		if header.MessagesHash != nil {
+			return fmt.Errorf("failure indicating commitment includes MessagesHash")
```

Comment:
Could also add cases for these two added failures to the `TestValidateBasic` test.

Reply:
Note that for other failure cases the tests currently exist in pool_test.go (i think i've put them there before adding the `ValidateBasic` function, and then forget moving those)
https://github.com/oasisprotocol/oasis-core/blob/64136dbb76df69f02d722a3b8ef919733589b51b/go/roothash/api/commitment/pool_test.go#L183-L193

Those should probably also be moved into `TestValidateBasic` and just a single "bad" failure indicating committeemen test could be left there.

Reply:
Done, please take another look.